### DISCUSSION TITLE
Why the bundlesize of formily so much more than react-hook-form

### DISCUSSION THREAD

**Discussion by nhattran998** [alibaba/formily#2720] [Q&A 2.x] [answered]
I check react-hook-form bundle size was [8.2 kb Minified + Gzip](https://bundlephobia.com/package/react-hook-form@7.22.5) 
And
[formily/react = 35.3kb Minified + Gzip](https://bundlephobia.com/package/@formily/react@2.0.7)
[formily/core = 29.1 kb Minified + Gzip](https://bundlephobia.com/package/@formily/core@2.0.7).

So someone can help me why we should use it in our projects with the bundle size will increase x8. Thanks <3

**Comment by janryWang** [comment, marked as answer]
check this https://github.com/alibaba/formily/pull/2737#issuecomment-1006614477
